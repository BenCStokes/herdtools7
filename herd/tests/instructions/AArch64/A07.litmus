AArch64 A07

(* Tests ldrh (register), SXTW *)
{ uint64_t 0:X0; 0:X1=x; uint64_t x; 0:X2=0; }

P0;
  LDRH W0, [X1, X2, SXTW];

exists (0:X0=0)
