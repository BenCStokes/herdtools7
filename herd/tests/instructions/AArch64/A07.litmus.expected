Test A07 Allowed
States 1
0:X0=0;
Ok
Witnesses
Positive: 1 Negative: 0
Condition exists (0:X0=0)
Observation A07 Always 1 0
Hash=f9e49f8c289ba262f75d16c89d629cdc

